scripts/benchmark_runner_windows.py

```diff
@@ -0,0 +1,169 @@
+"""Run a benchmark and return the results."""
+import argparse
+
+import subprocess
+import time
+import os
+import benchmark_utils as utils
+
+
+def create_extract_destinations(path):
+    """ Create the extract destination directories if they do not exist"""
+    if not os.path.exists(path):
+        os.mkdir(path)
+    os.chdir(path)
+    if not os.path.exists("base"):
+        os.mkdir("base")
+    if not os.path.exists("daily"):
+        os.mkdir("daily")
+
+
+def download_file(url, filename):
+    """ Download file from url and save it to filename"""
+    subprocess.run(['powershell', 'Invoke-WebRequest', '-Uri',
```

Comment:
Delegating this to PowerShell is not very elegant. I'm sure there is an easy way to download files purely in Python.